modules/preview-react/side-panel/lib/SidePanel.tsx

```diff
@@ -1,17 +1,16 @@
 /** @jsxRuntime classic */
 /** @jsx jsx */
 import * as React from 'react';
-import {styled, useIsRTL} from '@workday/canvas-kit-react/common';
-import {css, jsx, keyframes, CSSObject} from '@emotion/react';
-import {TertiaryButton, TertiaryButtonProps} from '@workday/canvas-kit-react/button';
-import {space, colors, depth} from '@workday/canvas-kit-react/tokens';
-import {transformationImportIcon} from '@workday/canvas-system-icons-web';
-import {Tooltip} from '@workday/canvas-kit-react/tooltip';
+import {createComponent, styled} from '@workday/canvas-kit-react/common';
```

Comment:
Can we split this refactor out into its own PR?

Reply:
Same with SidePanel